app/stac_api/admin.py

```diff
@@ -66,7 +66,17 @@ class ProviderInline(admin.TabularInline):
     }
 
 
-class CollectionLinkInline(admin.TabularInline):
+class LinkInline(admin.TabularInline):
+
+    def formfield_for_dbfield(self, db_field, request, **kwargs):
+        # make the hreflang field a bit shorter
```

Comment:
Maybe explain why it is useful to have this attribute name smaller/short? Not clear to me what this would change and why without seeing it.